### PR TITLE
Sync ORY/Oathkeeper charts to the latest version

### PR DESCRIPTION
**Pull request by mjakobczyk**
**Description**

Changes proposed in this pull request:

- update `oathkeeper` chart
- update `oathkeeper-maester` chart

**Related issue(s)**

Implements #9020

### CODE REVIEW

File: resources/ory/charts/oathkeeper/templates/ingress-api.yaml [hunk 1]
```diff
@@ -32,8 +32,8 @@ spec:
         {{- range .paths }}
           - path: {{ . }}
             backend:
-              serviceName: {{ $fullName }}-api
-              servicePort: http
+              serviceName: {{ $fullName }}-api # TODO: change or leave?
```
Comment (strekm) [inline resources/ory/charts/oathkeeper/templates/ingress-api.yaml:35]: ? seems to me you have some doubts

File: resources/ory/charts/oathkeeper/templates/ingress-proxy.yaml [hunk 2]
```diff
@@ -32,8 +32,8 @@ spec:
         {{- range .paths }}
           - path: {{ . }}
             backend:
-              serviceName: {{ $fullName }}-proxy
-              servicePort: http
+              serviceName: {{ $fullName }}-proxy # TODO: change or leave?
```
Comment (strekm) [inline resources/ory/charts/oathkeeper/templates/ingress-proxy.yaml:35]: same as above

File: resources/ory/charts/oathkeeper/templates/ingress-api.yaml (hunk 1 above)
Comment (mjakobczyk) [reply resources/ory/charts/oathkeeper/templates/ingress-api.yaml:35]: ORY chart uses `Ingress` resource with version `extensions/v1beta1`. We use `networking.k8s.io/v1` on our side. As a result, in our chart the marked key seems to be no longer needed (IDE prompts that this key is not expected). Should we then consider removing it or not?

File: resources/ory/charts/oathkeeper/templates/ingress-proxy.yaml (hunk 2 above)
Comment (mjakobczyk) [reply resources/ory/charts/oathkeeper/templates/ingress-proxy.yaml:35]: The same reply as above.

**Review by strekm — APPROVED** [review]
/meow

**Pull request merged**